components/twitter/actions/create-tweet/create-tweet.ts

```diff
@@ -17,7 +17,7 @@ export default defineAction({
       type: "string",
       label: "Text",
       description:
-        "Text of the Tweet being created. Required if `Media IDs` is not set.",
+        "Text of the Tweet being created. Required if `Media IDs` is not set. TEST",
```

Comment:
```suggestion
        "Text of the tweet being created. Required if `Media IDs` is not set. Test"
```

Reply:
Changed 'Tweet' to 'tweet' and removed all caps from 'TEST' to follow Google's Developer docs style guide and Dreyer's English